Release 4.1.0 - Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)

## Observed behaviour

I have this error on the `release-4.1` branch and the tag `v4.1.0` :
- `package javax.xml.bind.annotation does not exist`
- `cannot find symbol class XmlTransient`

```
[INFO] commons-entity-jpa 4.1.1-SNAPSHOT .................. SUCCESS [  0.263 s]
[INFO] asqatasun-api 4.1.1-SNAPSHOT ....................... FAILURE [  0.604 s]
[INFO] reference 4.1.1-SNAPSHOT ........................... SKIPPED
(...)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project asqatasun-api: Compilation failure: Compilation failure: 
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[38,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
```
same as issue #289  (develop branch) "Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)"

> ``` 
> java.xml.bind (JAXB)
>     Java 8 - OK
>     Java 9 - DEPRECATED
>     Java 10 - DEPRECATED
>     Java 11 - REMOVED
> ```
> source: https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist/52502208#52502208
> ```
> <dependency>
>   <groupId>javax.xml.bind</groupId>
>   <artifactId>jaxb-api</artifactId>
>   <version>2.3.1</version>
> </dependency>
> 
> ```
> source: https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j/52872160#52872160


## Expected behaviour
```
[INFO] asqatasun-api 4.1.1-SNAPSHOT ....................... SUCCESS
```

## Steps to reproduce the behaviour
```bash
git clone ...
cd Asqatasun
git checkout develop
mvn clean install
```

## Your configuration
```bash
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1, mixed mode, sharing)

Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.6, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: fr_FR, platform encoding: UTF-8
OS name: "linux", version: "5.3.0-45-generic", arch: "amd64", family: "unix"
```

This works well with jdk8 
using for example https://github.com/Asqatasun/asqa.mvn/blob/master/Asqatasun/jdk-8/Dockerfile



## Solution

same as :  
- #327 (develop branch)
- #289 "Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)"

```bash
git checkout origin/release-4.1
git cherry-pick 6fd316d
```

--- a/engine/asqatasun-api/pom.xml
+++ b/engine/asqatasun-api/pom.xml
@@ -67,5 +67,11 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
         </dependency>
+
+	<!-- Java Architecture for XML Binding (JAXB), Java 11+ support -->		
+	<dependency>
+		<groupId>javax.xml.bind</groupId>
+		<artifactId>jaxb-api</artifactId>
+	</dependency>
     </dependencies>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -132,6 +132,7 @@
         <jacoco-maven-plugin.version>0.8.1</jacoco-maven-plugin.version>
         <markdown-page-generator-plugin.version>2.1.0</markdown-page-generator-plugin.version>
         <yuicompressor-maven-plugin.version>1.5.1</yuicompressor-maven-plugin.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
     </properties>
     <repositories>
         <repository>
@@ -1788,6 +1789,13 @@
                 <groupId>com.ruleoftech</groupId>
                 <artifactId>markdown-page-generator-plugin</artifactId>
                 <version>${markdown-page-generator-plugin.version}</version>
+            </dependency>
+
+            <!-- Java Architecture for XML Binding (JAXB), Java 11+ support -->		
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>${jaxb-api.version}</version>
             </dependency>
         </dependencies>
     </dependencyManagement>

